changelog/fragments/add-componentconfig-flag.yaml

```diff
@@ -0,0 +1,26 @@
+entries:
+  - description: >
+      (ansible/v1, helm/v1) correctly support configuring the manager by file using the
+      `--config` flag. This flag was not added to either ansible-/helm-operator binary
+      when file support was originally added.
+    kind: bugfix
+    migration:
+      header: Add the manager config patch to config/default/kustomization.yaml
+      body: >
+        The scaffolded `--config` flag was not added to either ansible-/helm-operator binary
+        when config file support was originally added, so does not currently work.
+        The `--config` flag was added to both binaries to support configuration by file;
+        this method of configuration only applies to the underlying
+        [controller manager](https://pkg.go.dev/sigs.k8s.io/controller-runtime/pkg/manager#Manager),
+        not the operator as a whole. To configure the operator's Deployment with this config data,
+        add the following to config/default/kustomization.yaml:
```

Comment:
Do we really need that here? The solution is not just to bump the new image version for that? (default step to get the new version)

Reply:
I added a note about this being opt-in below. The kustomization.yaml change is necessary to make this work; the actual scaffold update comes from #4768 but this PR is more relevant to look at if a user is curious about why this bug fix was necessary IMO.